buddy-guide.md

```diff
@@ -0,0 +1,76 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Buddy Guide](#buddy-guide)
+- [Before the new hire starts](#before-the-new-hire-starts)
+- [Day 1](#day-1)
+- [Week 1](#week-1)
+- [End of Week 1](#end-of-week-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Buddy Guide
+
+As part of the onboarding process, each new hire will be allocated a buddy. A buddy is typically an experienced team member who will be
```

Comment:
yeah, tbh just lifted the entire existing buddy guide & think it will need a serious edit. My thinking was just to get a stub up so anyone who wants to take a pass it can